domain/tests/src/Functional/DomainDrushCommandsTest.txt

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Drupal\Tests\domain\Functional;
+
+//use Drupal\Tests\domain\Functional\DomainCommandTestBase;
+
+/**
+ * Tests the domain record creation API.
+ *
+ * @group domain
+ */
+class DomainDrushCommandsTest extends DomainCommandTestBase {
+
+  /**
+   * Show some info about the domain module environment.
+   */
+  public function testInfoDomains() {
+    if ($this->isWindows()) {
+      $this->markTestSkipped('Site-set not currently available on Windows.');
+    }
+    $this->setUpDrupal(1, true);
+
+    // Assure that a pending post-update is reported.
+    $this->drush('domain:info', [], ['format' => 'json']);
+    $out = $this->getOutputFromJSON('domain_access_entities');
+    $this->assertEquals('1', $out);
+  }
+
+
+  /**
+   * List the domains present on the system and some of their properties.
+   */
+  public function testListDomains() {
+    if ($this->isWindows()) {
+      $this->markTestSkipped('Site-set not currently available on Windows.');
+    }
+    $this->setUpDrupal(1, true);
+
+    // Assure that a pending post-update is reported.
```

Comment:
Same here. just reading it, I suspect that `getOutputFromJSON` is returning a boolean, which is why the test passes.